### DISCUSSION TITLE
Migrating create-react-app (CRA) application to turbo(pack)

### DISCUSSION THREAD

**Discussion by wowczarczyk** [vercel/turborepo#3496] [Help] [answered]
Hey guys!

I trying to find some materials on how such migration would be pulled off, but it seems there are none. Did anyone tried that already?

**Comment by wowczarczyk** [comment, marked as answer]
OK, actually I found out that turbopack can't be used without Next.js for the time being, so I guess I will just switch to Next.js first and take it from there.

**Reply by TaylorHorn** [reply]
Successor to Webpack, but can only be used with Next... Is there any plans around Turbopack being opened up to other Webpack dependent frameworks (React, Angular, etc...)?

**Reply by orenmizr** [reply]
and can it be configured to build correctly in a monorepo setup ?

**Reply by anthonyshew** [reply]
The Turbopack team is using Next.js as the proving ground for building to start things off. The eventual plan is to create a general purpose bundler if there ends up being interest!

**Reply by ThiefMaster** [reply]
an issue that can be followed to get updates regarding a standalone turbopack CLI would be great!
